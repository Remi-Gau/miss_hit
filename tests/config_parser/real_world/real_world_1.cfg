# Top level configuration for POTATO. Configuration deeper down will
# selectively enable MISS_HIT.
enable: 0
line_length: 100
file_length: 1000
copyright_entity: "Potato Inc."
copyright_entity: "Kittens AG"

# Naming rules. Disabled for now because people never did follow the style...
suppress_rule: "naming_classes"
suppress_rule: "naming_functions"
regex_class_name: "[A-Z]\w*"
regex_function_name: "[A-Z]\w*"
regex_method_name: "\w+"
regex_nested_name: "\w+"

# These rules produce too many alarms right now, and they cannot be auto-fixed
suppress_rule: "operator_after_continuation"

# These rules produce too many alarms right now (but they can be auto-fixed)
suppress_rule: "operator_whitespace"
suppress_rule: "end_of_statements"
