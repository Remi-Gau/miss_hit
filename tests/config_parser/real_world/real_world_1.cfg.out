MISS_HIT configuration file
  Global MH Disable
  Style configuration line_length
    Value: 100
  Style configuration file_length
    Value: 1000
  Style configuration copyright_entity
    Value: Potato Inc.
  Style configuration copyright_entity
    Value: Kittens AG
  Style application for naming_classes
    Enabled: False
  Style application for naming_functions
    Enabled: False
  Style configuration regex_class_name
    Value: [A-Z]\w*
  Style configuration regex_function_name
    Value: [A-Z]\w*
  Style configuration regex_method_name
    Value: \w+
  Style configuration regex_nested_name
    Value: \w+
  Style application for operator_after_continuation
    Enabled: False
  Style application for operator_whitespace
    Enabled: False
  Style application for end_of_statements
    Enabled: False
MISS_HIT Debug Summary: 1 file(s) analysed, everything seemes fine
